tests/dialects/test_mysql.py

```diff
@@ -83,6 +83,12 @@ def test_ddl(self):
                 "mysql": "CREATE TABLE IF NOT EXISTS industry_info (a BIGINT(20) NOT NULL AUTO_INCREMENT, b BIGINT(20) NOT NULL, c VARCHAR(1000), PRIMARY KEY (a), UNIQUE d (b), INDEX e (b))",
             },
         )
+        self.validate_all(
+            "CREATE TABLE t (id INT UNSIGNED)",
+            write={
+                "mysql": "CREATE TABLE t (id UNSIGNEDINT)",
```

Comment:
This is incorrect right?

Reply:
Does MySQL recognize UINT as a valid type?

Reply:
This test means "assert that `CREATE TABLE t (id INT UNSIGNED)` in MySQL is transpiled to `CREATE TABLE t (id UINT)` in MySQL", so yeah it's incorrect.